lib/models/src/main/java/org/sagebionetworks/repo/model/UnauthenticatedException.java

```diff
@@ -2,10 +2,12 @@
 
 public class UnauthenticatedException extends RuntimeException {
 
+	public static final String MESSAGE_USERNAME_PASSWORD_COMBINATION_IS_INCORRECT = "The provided username/password combination is incorrect";
+	
 	private static final long serialVersionUID = 11356746209283224L;
 
 	public UnauthenticatedException() {
-		super("Authenticated required.");
+		super(MESSAGE_USERNAME_PASSWORD_COMBINATION_IS_INCORRECT);

```

Comment:
Do you think it's safe to have a default reason string?  Might this not open the chance that someone throws UnauthenticatedException() for another reason (say, for a bad HMAC) not realizing that they are returning the wrong message?